code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -736,70 +726,79 @@
 	owner.update_health_hud() //update the healthdoll
 	owner.update_body()
 
-///Proc to change the value of the `owner` variable and react to the event of its change.
-/obj/item/bodypart/proc/set_owner(new_owner)
+/// Proc to change the value of the `owner` variable and react to the event of its change.
+/obj/item/bodypart/proc/update_owner(new_owner)
```

Comment:
At a glance it looks like you want to make this `SHOULD_NOT_OVERRIDE` rather than `SHOULD_CALL_PARENT`